nix_update/__init__.py

```diff
@@ -56,6 +58,12 @@ def parse_args(args: list[str]) -> Options:
         help="Regex to extract version with, i.e. 'jq-(.*)'",
         default="(.*)",
     )
+    parser.add_argument(
+        "-vk",
+        "--version-key",
```

Comment:
I think version-attr fits better with how nix language names things:

```suggestion
        "--version-attr",
```